README.md

```diff
@@ -21,6 +21,8 @@ The [Java Micobenchmarking Harness](http://openjdk.java.net/projects/code-tools/
 - JDK 1.7+
 - Eta
 
+Note: `eta` needs to be on the `PATH` (which is normally not the case). You can run `scripts/eta-install.sh` to clone/build/install the latest version of eta or you can edit/run `scripts/eta-select.sh` to add a local version of eta to the PATH.
```

Comment:
This is not necessarily true give my comment [here](https://github.com/typelead/eta-benchmarks/issues/10#issuecomment-455539445) - can you update this PR accordingly?